src/components/AuthUserState.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from "react";
+import { Auth, Hub } from "aws-amplify";
+import { Button } from "react-bootstrap";
+import { signIn, signOut } from '../store/actions/authentication'
+import { connect } from "react-redux";
+import { getUserState } from "../store/selectors"
```

Comment:
Can you explain to me how this `getUserState` API from `/store/selectors` get to be used because I only saw `const getLoggedInState` in it?